src/ocean/util/app/ext/TimerExt.d

```diff
@@ -27,6 +27,19 @@
 
 module ocean.util.app.ext.TimerExt;
 
+import ocean.util.log.Log;
```

Comment:
`Logger`, `Log` will be gone in v4 and is deprecated in v3 but v4.x.x is too much behind